client/app/calendar/tasks/hooks/useCreateTask.ts

```diff
@@ -0,0 +1,60 @@
+import { useQueryClient } from '@tanstack/react-query'
+import { useForm } from 'react-hook-form'
+import { useToast } from '@/_hooks'
+import { useAuthStore } from '@/_stores'
+import { isErrorWithMessage } from '@/_utils'
+import { useCreateTaskMutation } from './api'
+import { createTaskResolver, taskSchema } from '../schema'
+
+export const useCreateTask = () => {
+  const queryClient = useQueryClient()
+  const getHouseId = useAuthStore((state) => state.getHouseId)
```

Comment:
Please don't forget to change here after https://github.com/HAK2024/fairy-share/pull/76 is merged!

Reply:
fixed it!
https://github.com/HAK2024/fairy-share/pull/80/commits/719b7def73b3aacf854ef516fce29f7dae113f5d